repose-aggregator/docs/src/asciibinder/_templates/page.html.erb

```diff
@@ -11,6 +11,7 @@
   <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/css/bootstrap-theme.min.css">
   <link rel="stylesheet" href="/_stylesheets/asciibinder.css" >
   <link rel="stylesheet" href="/_stylesheets/repose.css" >
+  <link rel="stylesheet" href="/_stylesheets/release-note-categories.css" >
```

Comment:
Doing this here links this CSS to every generated page. Ideally, we would only link it to the release notes, but I could not find a way to get Asciidoctor to do that for me.